client/my-sites/themes/themes-banner/random-themes-banner.jsx

```diff
@@ -21,13 +17,29 @@ class RandomThemesBanner extends PureComponent {
 		const { banners } = props;
 		this.state = {
 			BannerComponent: sample( banners ),
```

Comment:
I'd rather set `this.BannerComponent` instead of state -- contrarily to `state`, it doesn't really change, so it's fine to have it as an instance var.

Also, can we set it in `componentDidMount` so we can drop the constructor? (We can set initial `isClient` state by simply using `state = { isClient: false }` after the `propTypes` definition.)